cms/migrations/0006_auto_20160421_1712.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-04-21 17:12
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def set_default_homepage_text(apps, schema_editor):
```

Comment:
Can you rename this migration to something human readable, ie `0006_change_homepage_title.py`?